transform/models/marts/repo_pushes_daily.sql

```diff
@@ -1,9 +1 @@
-SELECT 
-  repo_id, 
-  repo_name, 
-  DATE_TRUNC('day', event_date) AS date_day, 
-  COUNT(*) AS push_count, 
-  SUM(push_count) OVER(PARTITION BY repo_id ORDER BY date_day) AS cumul_push_count
-FROM {{ ref("stg_gharchive") }} 
-WHERE event_type = 'Push' 
-GROUP BY 1, 2, 3 
\ No newline at end of file
+{{ repo_event_by_period('Push', 'day') }}
```

Comment:
One final thing to fix: It's not great to have a capital letter e.g. `cumul_Push_count`.
In order to fix this, I'd suggest we lowercase the event types in the staging table.